src/python/datastore/data_types.py

```diff
@@ -1108,6 +1108,16 @@ def _pre_put_hook(self):
                        fuzz_target_job_key(self.fuzz_target_name, self.job))
 
 
+class FuzzStrategyProbability(Model):
+  """Mapping between fuzz strategies and probabilities with which they
+  should be selected
+
+  Calculated using a softmax multi-armed bandit probability model."""
+  strategy_name = ndb.StringProperty()
+  strategy_count = ndb.IntegerProperty()
+  strategy_probability = ndb.FloatProperty()
```

Comment:
Nit: just probability should be fine. The fact that it should be associated with the given strategy_name is implied.